Tasks/Common/ios-signing-common/ios-signing-common.ts

```diff
@@ -671,8 +675,12 @@ async function printFromPlist(itemToPrint: string, plistPath: string) {
     return printedValue;
 }
 
-function getProvisioningProfilePath(uuid: string): string {
-    return tl.resolve(getUserProvisioningProfilesPath(), uuid.trim().concat('.mobileprovision'));
+function getProvisioningProfilePath(uuid: string, provProfilePath?: string): string {
+    let profileExtension: string = '.mobileprovision';
+    if (provProfilePath) {
+        profileExtension = provProfilePath.substring(provProfilePath.lastIndexOf('.'));
```

Comment:
It would be good to test if profileExtension was assigned a empty value, and use the ".mobileprovision" extension in that case as well.